phpunit.xml

```diff
@@ -8,7 +8,7 @@
          convertWarningsToExceptions="true"
          processIsolation="false"
          syntaxCheck="false"
-         bootstrap="test/bootstrap.php"
+         bootstrap="./vendor/autoload.php"
```

Comment:
I'm not sure this is a good idea. The bootstrap.php of phpunit is more than just autoloading. The file you just removed also has a line that loads a file functions.php for example.

Reply:
@samnela is correct here. However, if we're just relying on composer's autoload we can drop this line since PHPUnit is installed via composer (which means that the autloader is already there)